OpenRA.Mods.Common/Traits/World/MPStartLocations.cs

```diff
@@ -89,7 +130,7 @@ static CPos ChooseSpawnPoint(World world, List<CPos> available, List<CPos> taken
 			if (available.Count == 0)
 				throw new InvalidOperationException("No free spawnpoint.");
 
-			var n = taken.Count == 0
+			var n = taken.Count == 0 || !world.WorldActor.Trait<MPStartLocations>().separateTeamSpawns
```

Comment:
This is already in the `MPStartLocations`, so ` || !separateTeamSpawns` is sufficient.

Reply:
but it's in static method where bool separateTeamSpawns isn't static